### PR TITLE
Adds Dynatrace metrics

### PR DESCRIPTION
**Pull request by krishvoor**
This PR adds Dynatrace metrics, adds panels for
1) OneAgent CPU/Mem
2) ActiveGate CPU/Mem
3) Opentelemetry CPU/Mem

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by krishvoor** [comment]
Hi @mukrishn 👋 Can you ptal?

**Comment by krishvoor** [comment]
Corresponding metrics were captured here: https://snapshots.raintank.io/dashboard/snapshot/9097aD1U7WAoLBttllrCDDtz3GK7eEdC?orgId=2

File: templates/General/ocp-performance.jsonnet [hunk 1]
```diff
@@ -184,6 +184,49 @@ local stackroxMem = genericGraphLegendPanel('Top 25 stackrox container RSS bytes
   )
 );
 
+// Dynatrace
```
Comment (mukrishn) [inline templates/General/ocp-performance.jsonnet:187]: @krishvoor As discussed, let keep this Hypershift related metrics in to hypershift-performance dash.

Comment (krishvoor) [reply templates/General/ocp-performance.jsonnet]: @mukrishn I made the changes as follows:
1) OneAgent Pods Will be captured per cluster serving nodes
2) ActiveGate  & Opentelemetry Pods will be captured at the Management Cluster metrics level

**Review by mukrishn — APPROVED** [review]
lgtm

**Comment by krishvoor** [comment]
Grafana Report: https://snapshots.raintank.io/dashboard/snapshot/7CDCJ4vM6y68L7k03ouV73f30NR3XQnu 
Make Build Logs: [make_build.log](https://github.com/cloud-bulldozer/performance-dashboards/files/13257259/make_build.log)

**Comment by krishvoor** [comment]
Hi @vishnuchalla @mukrishn 👋  ptal

File: dittybopper/deploy.sh [hunk 2]
```diff
@@ -41,7 +41,7 @@ END
 export PROMETHEUS_USER=internal
 export GRAFANA_ADMIN_PASSWORD=admin
 export GRAFANA_URL="http://admin:${GRAFANA_ADMIN_PASSWORD}@localhost:3000"
-export SYNCER_IMAGE=${SYNCER_IMAGE:-"quay.io/cloud-bulldozer/dittybopper-syncer:latest"} # Syncer image
+export SYNCER_IMAGE=${SYNCER_IMAGE:-"quay.io/krvoora_ocm/dittybopper-syncer:latest"} # Syncer image
```
Comment (vishnuchalla) [inline dittybopper/deploy.sh:44]: Can we change this to official `cloud-bulldozer` image?

**Review by vishnuchalla — APPROVED** [review]
lgtm

**Comment by krishvoor** [comment]
@vishnuchalla @mukrishn if this is looking good can you merge this?

File: Makefile [hunk 3]
```diff
@@ -1,5 +1,8 @@
-JSONNET := https://github.com/google/go-jsonnet/releases/download/v0.20.0/go-jsonnet_0.20.0_Linux_x86_64.tar.gz
-JB = https://github.com/jsonnet-bundler/jsonnet-bundler/releases/latest/download/jb-linux-amd64
+ARCH := $(shell arch)
```
Comment (smalleni) [inline Makefile:1]: Can you make updates to the makefile a separate PR please since this is about adding dynatrace metrics.

**Review by smalleni — CHANGES_REQUESTED** [review]
See comment

File: Makefile (hunk 3 above)
Comment (krishvoor) [reply Makefile:1]: Sure, will spin up another PR for Makefile related. changes.

Comment (krishvoor) [reply Makefile]: Bifurcated Makefile-related changes to this [PR](https://github.com/cloud-bulldozer/performance-dashboards/pull/93)

**Review by mukrishn — APPROVED** [review]

**Review by chentex — APPROVED** [review]

**Review by smalleni — APPROVED** [review]
